great-idea/index.html

```diff
@@ -17,7 +17,77 @@
   <body>
 
     <!-- Copy and paste your HTML from the first UI project here -->
+    <nav>
+      <a href='services.html'>Services</a>
+      <a href='#'>Product</a>
+      <a href='#'>Vision</a>
+      <a href='#'>Features</a>
+      <a href='#'>About</a>
+      <a href='#'>Contact</a>
       
+      <a href="index.html">
+        <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+      </a>
+    </nav>
+  
+    <header class="index-header">
+      <h1>
+        Innovation<br>
+        On<br>
+        Demand<br>
+        <button>Get Started</button>
+      </h1>
+
+      <img src="img/header-img.png" alt="Image of a code snippet.">
+    </header>
+
+    <hr>
+
+    <section class='top-content'>
+      <div>
+        <h2>Features</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+
+      <div>
+        <h2>About</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+    </section>
+    <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+  
+    <section class='bottom-content'>
+      <div>
+        <h2>Services</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+
+      <div>
+        <h2>Product</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+
+      <div>
+        <h2>Vision</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+    </section>
+    
+    <hr>
+
+    <address>
+      <h3>Contact</h3>
+      123 Way 456 Street<br>
+      Somewhere, USA<br>
+      <br>
```

Comment:
Excellent HTML organization throughout, try adding anchor tags to the phone number and email with their respective href attributes (bonus)